syntax = "proto3";

message ServerLink {
    string server_str = 1;
}
message DataBaseNotify{
    string db_ip = 1;
    uint32 db_port = 2;
}

message ServerMessageRegister{
    repeated string message_str = 1;
}